app/views/menu.erb

```diff
@@ -0,0 +1,31 @@
+
+<div class="">
+  <h1><%= @menu_name %></h1>
+  <% @menu.each do |product| %>
+    <% if product[:catagory] == 'Starter' %>
```

Comment:
I don't mind this syntax, but I prefer `product.category` Same thing in all lines below where you want to display the attribute values of the `product`